stdlib/Profile/src/Profile.jl

```diff
@@ -55,17 +55,26 @@ function init(; n::Union{Nothing,Integer} = nothing, delay::Union{Nothing,Real}
     if n === nothing && delay === nothing
         return Int(n_cur), delay_cur
     end
-    nthreads = Sys.iswindows() ? 1 : Threads.nthreads() # windows only profiles the main thread
-    nnew = (n === nothing) ? n_cur : n * nthreads
+    nnew = (n === nothing) ? n_cur : n
     delaynew = (delay === nothing) ? delay_cur : delay
     init(nnew, delaynew)
 end
 
 function init(n::Integer, delay::Real)
     nthreads = Sys.iswindows() ? 1 : Threads.nthreads() # windows only profiles the main thread
-    status = ccall(:jl_profile_init, Cint, (Csize_t, UInt64), n * nthreads, round(UInt64,10^9*delay))
+    sample_size_bytes = 1 # what should this be?
```

Comment:
@vtjnash given the approval, I just wanted to check that you mean this is correct? I'll update if so

Reply:
```suggestion
    sample_size_bytes = sizeof(Ptr) # == Sys.WORD_SIZE / 8
```